modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/NoUnnecessaryRebalancesTest.java

```diff
@@ -115,28 +113,20 @@ public void testNoRebalancesOnCacheCreation() throws Exception {
     /**
      * Wrapper of communication spi to detect on which cache groups rebalances were happened.
      */
-    public static class SpecialSpi extends TestRecordingCommunicationSpi {
+    public static class RebalanceTrackingSpi extends TestRecordingCommunicationSpi {
```

Comment:
Can we use `#record` and `#recordedMessages` functionality of the `TestRecordingCommunicationSpi` to check messages?

Reply:
I've fixed this as you suggested.